include/wx/webrequest.h

```diff
@@ -188,6 +188,10 @@ class WXDLLIMPEXP_NET wxWebRequest
 
     wxWebRequestHandle GetNativeHandle() const;
 
+    void DisablePeerVerify(bool disable = true);
+
+    bool IsPeerVerifyDisabled();
```

Comment:
Somewhat more important here:
```suggestion
    bool IsPeerVerifyDisabled() const;
```